scanners/zap-automation-framework/integration-tests/automation-framework-configMap.yaml

```
@@ -0,0 +1,219 @@
+# SPDX-FileCopyrightText: the secureCodeBox authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: "zap-automation-framework-config"
```

would be more consistent as
```suggestion
  name: "zap-automation-juicehop"
```

the number in the file names here are also not really required anymore